From f331b7fd770c3d284120c445d0571f9b2cd52a57 Mon Sep 17 00:00:00 2001
From: Kody A Kantor <kody.kantor@gmail.com>
Date: Tue, 8 May 2018 20:04:03 +0000
Subject: [PATCH] MANTA-2980 electric-moray should report moray shard for
 operations like 'putObject' Reviewed by: Jordan Hendricks
 <jordan.hendricks@joyent.com> Approved by: Jordan Hendricks
 <jordan.hendricks@joyent.com>

---
 lib/server.js | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/server.js b/lib/server.js
index 7f6d12c..c019a11 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -749,6 +749,9 @@ function putObject(options) {
                 if (pErr) {
                     rpc.fail(pErr);
                 } else {
+                    // Add shard information to the response.
+                    meta._node = node;
+
                     rpc.write(meta);
                     rpc.end();
                 }
-- 
2.21.0

